tests/testthat/test-i18n.R

```diff
@@ -172,3 +172,54 @@ test_that("load resource bundles", {
   expect_equal(obj$t("login:username"), "Username")
   expect_equal(obj$t("login:username", language = "fr"), "username")
 })
+
+
+test_that("simple fallback", {
+  obj <- i18n(NULL, default_namespace = "common", fallback = "en")
+  obj$add_resource_bundle("en", "common",
+                          traduire_file("examples/structured/en-common.json"))
+  obj$add_resource_bundle("en", "login",
+                          traduire_file("examples/structured/en-login.json"))
+  obj$add_resource_bundle("fr", "common",
+                          traduire_file("examples/structured/fr-common.json"))
+  expect_equal(obj$t("hello", language = "en"), "hello world")
+  expect_equal(obj$t("hello", language = "fr"), "salut le monde")
+  expect_equal(obj$t("hello", language = "ko"), "hello world")
```

Comment:
The fallback covers missing keys too?

Worth adding a test to show how that behaves?

Reply:
done